src/common/cloudinary.ts

```diff
@@ -53,3 +53,8 @@ const avatarPreset = 'avatar';
 
 export const uploadAvatar = (userId: string, stream: Readable) =>
   uploadFile(`${avatarPreset}_${userId}`, avatarPreset, stream);
+
+const postPreset = 'post';
+
+export const uploadPostImage = (postId: string, stream: Readable) =>
```

Comment:
@idoshamun Is there anything here we need to keep in mind regards to cloudinary rules?
Having this cost issue in mind.

Reply:
I agree, also that in v6 we will allow everyone to upload images which we must monitor the usage.

Reply:
@rebelchris this is a great catch! @sshanzel please talk to Denis to understand what transformations and upload preset he applies to make sure we're aligned on this. Please tag me on the relevant lines once you apply that. Until that let's block the PR

Reply:
Code used as inspiration based on the discussion:
https://github.com/dailydotdev/gatekeeper/blob/48842a882167a6e61d82a6e1e74e8c191c73d8b8/src/common/cloudinary.ts#L17

Relevant change:
https://github.com/dailydotdev/daily-api/pull/1262/files#diff-44c49f4dbf41a19d0e49a6f2c185d26132919a5e2b76d00c0a75fa7de72d9370R59-R84

Relevant Slack discussion:
https://dailydotdev.slack.com/archives/C03C3JZPRU4/p1683634554381399?thread_ts=1683633804.026429&cid=C03C3JZPRU4